src/scipp/core/cpp_classes.pyi

```diff
@@ -333,12 +333,9 @@ class DataArray:
     def assign_attrs(
         self, attrs: dict[str, Variable] | None = None, /, **attrs_kwargs: Variable
     ) -> DataArray: ...
-    def assign_coords(
-        self, coords: dict[str, Variable] | None = None, /, **coords_kwargs: Variable
-    ) -> DataArray: ...
-    def assign_masks(
-        self, masks: dict[str, Variable] | None = None, /, **masks_kwargs: Variable
-    ) -> DataArray: ...
+
+    def assign_coords(self, arg0: str | Sequence[str]) -> DataArray: ...
+    def assign_masks(self, arg0: str | Sequence[str]) -> DataArray: ...
```

Comment:
I think something went wrong with the changes in this file. I don't think these lines should have been changed, but only new lines added for the `Bins` class?